### PR TITLE
Feature/submodule end interface

### PR DESCRIPTION
**Pull request by ruckim**

### CODE REVIEW

File: packages/tatum-ada/src/fungible/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Currency, getERC20TransactionsByAddress as getERC20TransactionsByAddressCore, Sort } from '@tatumio/tatum-core'
```
Comment (chrto) [inline packages/tatum-ada/src/fungible/index.ts]: Does ADA support fungible tokens?

File: packages/tatum-ada/src/multiToken/index.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+export const getMultiTokensBatchBalance = async (contractAddress: string, address: string, tokenIds: string) => {
+  return getMultiTokensBatchBalanceCore(Currency.ADA, contractAddress, address, tokenIds)
+}
+export const getMultiTokenTransaction = async (txId: string) => {
+  return getMultiTokenTransactionCore(Currency.ADA, txId)
+}
+export const getMultiTokenMetadata = async (contractAddress: string, tokenId: string) => {
+  return getMultiTokenMetadataCore(Currency.ADA, contractAddress, tokenId)
+}
```
Comment (chrto) [inline packages/tatum-ada/src/multiToken/index.ts]: Does ADA support multi tokens (ERC-1155)?

**Review by chrto — CHANGES_REQUESTED** [review]
Pls. check which functionality is supported for which blockchain.
https://tatum.io/apidoc.php

File: packages/tatum-ada/src/fungible/index.ts (hunk 1 above)
Comment (ruckim) [reply packages/tatum-ada/src/fungible/index.ts]: No according to docs, removed.

File: packages/tatum-ada/src/multiToken/index.ts (hunk 2 above)
Comment (ruckim) [reply packages/tatum-ada/src/multiToken/index.ts]: No according to docs, removed.

**Pull request merged**